core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -602,9 +601,6 @@ public void configure() throws ConfigurationException {
 			if (getListener() instanceof JdbcFacade) {
 				((JdbcFacade)getListener()).setTransacted(isTransacted());
 			}
-			if (getListener() instanceof JMSFacade) {
-				((JMSFacade)getListener()).setTransacted(isTransacted());
-			}
```

Comment:
Ik denk dat in plaats van dit weghalen, we een interface moeten toevoegen die door zowel JdbcFacade als JMSFacade geimplementeerd kan worden.
We hebben nu `IXAEnabled` die de method `isTransacted()` specificeert, we moeten denk een interface toevoegen die dit spiegelt met een method `setTransacted(boolean)`.

Dan hebben we ook gelijk 1 JDBC-dependency minder hier.

Interface naam... `EnableXA` misschien?